master/custom/factories.py

```diff
@@ -363,6 +363,34 @@ class NoBuiltinHashesUnixBuildExceptBlake2(UnixBuild):
     factory_tags = ["no-builtin-hashes-except-blake2"]
 
 
+class FedoraBuild(UnixBuild):
+    # Build on Fedora or RHEL.
+    #
+    # Try to be as close as possible to the Fedora specfile used to build
+    # the RPM package:
+    # https://src.fedoraproject.org/rpms/python3.10/blob/rawhide/f/python3.10.spec
+    configureFlags = [
+        "--with-pydebug",
+        "--with-platlibdir=lib64",
```

Comment:
This only makes sense on 64bit arches.

Reply:
All the Fedora and RHEL buildbots are 64 bits.